landoapi/workers/landing_worker.py

```diff
@@ -414,6 +415,10 @@ def run_job(
             LandingJobAction.LAND, commit_id=commit_id, commit=True, db=db
         )
 
+        mots_path = Path(hgrepo.path) / "mots.yaml"
+        if mots_path.exists():
+            job.set_landed_reviewers(mots_path)
```

Comment:
May need a catch-all exception here, in case someone breaks `mots.yaml` since we don't have an automated check that prevents someone from landing a corrupted version of mots.yaml at this time.